### ISSUE TITLE
incorrect/insufficient parentage for two sperm CC terms

### ISSUE THREAD

**Issue by krchristie** [geneontology/go-ontology#12186]
Hi Paola,

I was doing some annotations today and came across a couple terms that look incorrectly or insufficiently placed to me.

GO:0035686 - sperm fibrous sheath (currently is_a "ciliary part")

GO:0001520 - outer dense fiber (currently is_a "axoneme part")

Both of these are specialized structures specific to sperm flagella, so the is_a "ciliary part" relationship for the first term is fine but not sufficient. The is_a "axoneme part" relationship for "outer dense fiber" (ODF) is incorrect. The ODFs are separate structures that surround the sperm axoneme. The diagram in this paper (http://www.ncbi.nlm.nih.gov/pubmed/21586547) is helpful.

So, looking at the existing relationships for the term "sperm principal piece" these are the relationships that I think these two terms should have:
is_a "ciliary part (GO:0044441)"
is_a "sperm part (GO:0097223)"
part_of "sperm flagellum (GO:0036126)"

thanks,

Karen

#paolaroncaglia

**Comment by krchristie** [comment]
Also, I think the definition of "outer dense fiber" requires revision.

GO:0001520 - outer dense fiber
Current Def: Structure or material found in the flagella of mammalian sperm that surrounds each of the nine microtubule doublets, giving a 9 + 9 + 2 arrangement rather than the 9 + 2 pattern usually seen. These dense fibers are stiff and noncontractile.

I don't see any indication from the paper I cited earlier that this is even correct that the axoneme of mammalian sperm is 9+9+2 instead of 9+2 (there is variability in many species, but primarily not in mammals, see additional reference below) In addition, the ODFs are not confined to mammalian sperm.

Inaba. 2007. Molecular basis of sperm flagellar axonemes: structural and evolutionary aspects.
PMID:17363437 

Quote: The morphology of sperm cell varies among organisms and atypical axonemal structures are observed in sperm of some organisms: 9 + 9 + 2 (caddis fly, Neuronia sp.), 9 + 9 + 3 (black fly, Odagmia pontina), 9 + 3 (spider, Pholcus phalangioides), 9 + 7 (caddisfly, Polycentropus sp.), 9 + 0 (myzostomid worm, Myzostoma cirrifera), 9 + 9 + 0 (mayfly, Chloeon dipteron), 9 + 1 (flatworm Mecynostomum auritum), 9 + 9 + 1 (mosquito Culex sp.), 12 + 0 (proturan insect, Acerentulus tradgardhi), 14 + 0 (proturan insect, Acerentomon majus). In some cases, the axonemes are present but they are composed only with numerous numbers of doublet microtubules, such as in the cecidomyiid Monarthropalpus buxi.
http://onlinelibrary.wiley.com/doi/10.1196/annals.1389.017/full

**Comment by paolaroncaglia** [comment]
Hi Karen,

Thanks, I'll look into this. 
Afraid your mention of @paolaroncaglia didn't work because you used a different punctuation, but I found this all the same ;-)

Best,
Paola

**Comment by krchristie** [comment]
Thanks Paola,

Sorry I forgot the correct punctuation for tagging in GitHub. Glad you found it anyway :)

-K

**Comment by paolaroncaglia** [comment]
Hi Karen,

To start with, I added the link 'sperm fibrous sheath' part_of 'sperm flagellum'.
(The link 'sperm fibrous sheath' is_a 'sperm part' will be inferred from the one above.)

I'll look into outer dense fibres next.

**Comment by paolaroncaglia** [comment]
Hi again Karen,

Here is the updated stanza for outer dense fiber. Details below the stanza:

[Term]
id: GO:0001520
name: outer dense fiber
namespace: cellular_component
def: "Structure or material found in the flagella of mammalian sperm that surrounds the nine microtubule doublets. These dense fibers are stiff and noncontractile." [GOC:cilia, GOC:krc, ISBN:0824072820, PMID:21586547]
synonym: "outer dense fibre" EXACT []
is_a: GO:0044441 ! ciliary part
relationship: only_in_taxon NCBITaxon:40674 ! Mammalia
relationship: part_of GO:0036126 ! sperm flagellum

Definition: I suspect that the bit about the '9 + 9 + 2 arrangement' reflects the 9 ODFs + (9 + 2) axoneme. Possibly, at a time when it wasn't clear if the ODFs were true axonemal structure or not, this resulted in the ambiguity. The new def should solve this.
However, it would be really helpful to capture if recent research has clarified the nature of the ODFs. 'Structure or material' is quite poor and prevents us from giving ODF a real is_a parent (other than the 'structural' is_a ciliary part). Does http://www.ncbi.nlm.nih.gov/pubmed/21586547 elaborate on this, or do you have insights to share from other papers please?

I changed the parentage as you suggested, and I added a taxon rule for only in mammals. Until we know to the contrary, this will hold true and reflect the def.

**Comment by krchristie** [comment]
Great. Thanks Paola. Sounds like a good reflection of current knowledge.

-Karen

**Comment by paolaroncaglia** [comment]
@krchristie 
outer dense fiber = "Structure or material found in the flagella of mammalian sperm that surrounds the nine microtubule doublets. These dense fibers are stiff and non contractile."
Do we have any info on the more precise nature of such 'structure or material'? 
'Structure or material' is quite poor and prevents us from giving ODF a real is_a parent (other than the 'structural' is_a ciliary part). Does http://www.ncbi.nlm.nih.gov/pubmed/21586547 elaborate on this, or do you have insights to share from other papers please?
If not, I'll ask John.
Thanks!
Paola

**Comment by paolaroncaglia** [comment]
Hi @JohnvanDam,

We’re looking at GO:0001520 outer dense fiber. Currently, this is defined as "Structure or material found in the flagella of mammalian sperm that surrounds the nine microtubule doublets. These dense fibers are stiff and noncontractile.”, and has the only parent is_a ciliary part.
Do you have insights into the nature of outer dense fibers? What they are made of…? I’m looking for information that could help us add a more informative parent than just ‘ciliary part’. This is less of a priority than other tickets, but while I’m here…

Thanks!

**Comment by JohnvanDam** [comment]
Hi Paola,

I've just performed a quick literature search and am surprised about how relatively little research there is on ODF.
Here is one paper which has a decent introduction on ODF: http://molehr.oxfordjournals.org/content/5/7/627.full

From that abstract: "Our results suggest that human ODF consist of about 10 major and of at least 15 minor proteins, where all major proteins are ODF1, ODF2 or ODF2-related proteins."
This is from 1999.

Around these fibers you have the fibrous sheath comprising of smaller fibers that run parallel to the ODFs and axoneme and some ring shaped fibers that are perpendicular to the axoneme. But it is the ODFs that you can see quite clearly in EM photographs:
![OFDs in sperm](http://www.reproduction-online.org/content/135/3/303/F3.large.jpg)

I've checked "Sperm Annulus" GO:0097227, sperm connecting piece (GO:0097224), sperm fibrous sheath (GO:0035686), and they have in addition is_a relationships with:
cell projection part (GO:0044463)
organelle part (GO:0044422)

I guess those make sense.

I hope this helps

**Comment by JohnvanDam** [comment]
Hi Paola,

Strangely in amigo I just found that "sperm part" has an "is_a" child relationship with ODF (which would be correct), but the ODF entry doesn't list "sperm part" as an "is_a" parent term.

Am I missing something?

**Comment by paolaroncaglia** [comment]
Hi John,

Thanks for your insights into ODF, which I'll look at more closely as soon as I have a chance.
As for AmiGO, it's probably just a small delay issue - I made the changes 2 days ago - but I'll check tomorrow.

Thanks!

**Comment by dosumis** [comment]
Some possible classifications:

id "GO:0099512"
label "supramolecular fiber"
definition "A polymer consisting of an indefinite number of protein or protein complex subunits that have polymerised to form a fiber-shaped structure."

Or, of you consider these fibers to be part of the cytoskeleton:

id "GO:0099513"
"polymeric cytoskeletal fiber"
definition "A component of the cytoskeleton consisting of a homo or heteropolymeric fiber constructed from an indeterminate number of protein subunits."

![image](https://cloud.githubusercontent.com/assets/112839/13847089/373185dc-ec43-11e5-98bc-7cb8c08da7be.png)

**Comment by krchristie** [comment]
I've looked a couple different times and have found it very hard to find much info about the structure or composition of ODFs. This recent paper:

Wang X, Wei Y, Fu G, Li H, Saiyin H, Lin G, Wang Z, Chen S, Yu L. Tssk4 is
essential for maintaining the structural integrity of sperm flagellum. Mol Hum
Reprod. 2015 Feb;21(2):136-45. doi: 10.1093/molehr/gau097. Epub 2014 Oct 31.
PubMed PMID: 25361759.

refers to them as cytoskeletal elements though:

"In sperm tail, the outer dense fibers (ODFs) are prominent cytoskeletal elements that contribute to the distinctive morphological features of the motile apparatus of the sperm (Oko and Clermont, 1990). Previous studies have indicated that ODFs function in improving the bending torque of the tail (Lindemann, 1996) and/or protecting the sperm tail against shear forces during epididymal transportation and ejaculation (Baltz et al., 1990). The ODFs have been reported to be highly phosphorylated (Chaudhry et al., 1995; NagDas et al., 2002; Rosales et al., 2008). Odf2 was identified as a major protein component of ODFs in the mammalian sperm tail (Brohmann et al., 1997; Hoyer-Fender et al., 1998) localized to the medulla and cortex and to the connecting piece (Schalles et al., 1998). Odf2 can interact with cdk5, which upon activation can phosphorylate Odf1 (Rosales et al., 2008). Two Odf2 mouse models have been reported. One group generated chimaeras using a gene trap RO072 embryonic stem cell line, which resulted in an insertion into exon 9 of Odf2 gene, but this modification displayed preimplantation lethality (Salmon et al., 2006). The other group made Odf2 knockout mice using XL169 embryonic stem (ES) cells and the male mice with a high percentage chimaerism were infertile, whereas mice of low-medium percentage chimaerism were fertile. More than half of the epididymal sperm displayed bent tails, with one or more entire ODFs missing and the absence of one or more axonemal microtubule doublets (Tarnasky et al., 2010)."

**Comment by paolaroncaglia** [comment]
Thank you all for your feedback. This is very interesting (a polite way to say I’m opening a can of worms). Many elements point to this parent for ‘outer dense fiber’:

id "GO:0099513"
"polymeric cytoskeletal fiber"
definition "A component of the cytoskeleton consisting of a homo or heteropolymeric fiber constructed from an indeterminate number of protein subunits."

However, I’m reluctant to make this link because cytoskeleton is_a ‘intracellular non-membrane-bounded organelle’, while we explicitly made cilium is_a generic ‘organelle’ taking it out from under intracellular. @dos: ATM, polymeric cytoskeletal fiber has no links to cytoskeleton, which apparently clashes with its definition. Did you do this on purpose? 
Should we revise the placement of cytoskeleton and make it a generic ‘non-membrane-bounded organelle’? Its def says 

“Any of the various filamentous elements that form the internal framework of cells, and typically remain after treatment of the cells with mild detergent to remove membrane constituents and soluble components of the cytoplasm. The term embraces intermediate filaments, microfilaments, microtubules, the microtrabecular lattice, and other structures characterized by a polymeric filamentous nature and long-range order within the cell. The various elements of the cytoskeleton not only serve in the maintenance of cellular shape but also have roles in other cellular functions, including cellular movement, cell division, endocytosis, and movement of organelles.”

Axonemes make cilia move; they have_part microtubules, which are part_of cytoskeleton. We’re not breaking anything, formally, but shouldn’t the axoneme be considered cytoskeleton? I can’t remember that we discussed this previously. 

As for ODFs, many things in the def of cytoskeleton seem to apply to them quite fittingly. But for the moment, I’ll stick with ODF is_a generic ‘supramolecular fiber’. I have revised the term, here's the updated stanza:

[Term]
id: GO:0001520
name: outer dense fiber
namespace: cellular_component
def: "A supramolecular fiber found in the flagella of mammalian sperm that surrounds the nine microtubule doublets. These dense fibers are stiff and noncontractile. In human, they consist of about 10 major and at least 15 minor proteins, where all major proteins are ODF1, ODF2 or ODF2-related proteins." [GOC:cilia, GOC:krc, ISBN:0824072820, PMID:10381817, PMID:21586547, PMID:25361759]
synonym: "outer dense fibre" EXACT []
is_a: GO:0044441 ! ciliary part
is_a: GO:0099512 ! supramolecular fiber
relationship: only_in_taxon NCBITaxon:40674 ! Mammalia
relationship: part_of GO:0036126 ! sperm flagellum

**Comment by paolaroncaglia** [comment]
As an aside, @cmungall  any reason why GO:1990732 ‘pyrenoid’ is not intracellular? Bit weird, it should be...

**Comment by paolaroncaglia** [comment]
A question for @tonysawfordebi please.
3 days ago I added the link 
GO:0001520 outer dense fiber part_of GO:0036126 sperm flagellum
How long does it take for QuickGO to record changes? The added link is not yet listed in the Change Log panel, and, more importantly, it’s not shown in the Ancestor Chart. The QuickGO display is also consumed by AmiGO in the Graph Views (and that is also not showing the change). I can see ontology changes live in OBO-Edit or Protege, but my external collaborators, who use AmiGO or QuickGO, can’t.
But actually, I just realised that new terms created on the same date as the above link or thereabout are still not shown in QuickGO… GO:1990930 RNA N1-methyladenosine dioxygenase activity, GO:1990931 RNA N6-methyladenosine dioxygenase activity

**Comment by dosumis** [comment]
> @dos: ATM, polymeric cytoskeletal fiber has no links to cytoskeleton, which apparently clashes with its definition. Did you do this on purpose? 

Should be 'part of'. Is now.  (Planning a logical definition, but waiting on a PATO term request before I can add it).

>   I’m reluctant to make this link because cytoskeleton is_a ‘intracellular non-membrane-bounded organelle’, while we explicitly made cilium is_a generic ‘organelle’ taking it out from under intracellular.

I don't follow.  Here's the relevant def:
*Intracellular:* "The living contents of a cell; the matter contained within (but not including) the plasma membrane, usually taken to exclude large vacuoles and masses of secretory or ingested material. In eukaryotes it includes the nucleus and cytoplasm."

Cilia are not intracellular because they include plasma membrane.  But that doesn't mean they don't have parts that are intracellular.  Based on the definition, I'd say that all the parts or cilia that are not part of the plasma membrane are intracellular - and that includes the ODFs & axoneme. Classifying cytoskeleton as an organelle may be a bit of a stretch, but there's not rule to say that organelles can't overlap each other.  So, I think it's fine to have both ODFs and axonemes as part of the cytoskeleton (and by inference intracellular).  No need to change the classification of cytoskeleton.

**Comment by dosumis** [comment]
BTW - you can rely on the reasoner to add (& tag) the 'ciliary part' classification.

**Comment by paolaroncaglia** [comment]
Read the whole thread again. Left to do for this ticket:

- move GO:1990732 ‘pyrenoid’ under ‘intracellular non-membrane-bounded organelle’
- move GO:0001520 ‘outer dense fiber’ under ‘polymeric cytoskeletal fiber’
- add link ‘axoneme’ part_of cytoskeleton

Then close.

**Comment by paolaroncaglia** [comment]
All done as above.
